src/components/tile-implementations/federal-funds-rate/FederalFundsRateTile.tsx

```
@@ -1,24 +1,183 @@
+        filterDate.setMonth(now.getMonth() - 3);
+        break;
+      case '6M':
+        filterDate.setMonth(now.getMonth() - 6);
+        break;
+      case '1Y':
+        filterDate.setFullYear(now.getFullYear() - 1);
+        break;
+      case '5Y':
+        filterDate.setFullYear(now.getFullYear() - 5);
+        break;
+      case 'Max':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)